layouts/default/65_ansi_split_bs/default_65_ansi_split_bs/keymap.c

```diff
@@ -0,0 +1,24 @@
+#include QMK_KEYBOARD_H
```

Comment:
@fauxpark did we want/need a header here (pretty sure, yes)

Reply:
I kind of prefer them without a header, since they are, well, default layouts. Nobody really "owns" them, IMO. But I guess I don't mind.
Personally I would just add them with no name attached like I've done for keymap_extras.